django3_codemods/config_tools/tox_modernizer.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import re
+from configparser import ConfigParser, NoSectionError
+
+TOX_SECTION = "tox"
+ENV_SECTION = "envlist"
+TEST_ENV_SECTION = "testenv"
+TEST_ENV_DEPS = "deps"
+PY_38 = "py{38}"
```

Comment:
We'll probably want to be using `py{35,38}` for any repos still used in services running on 3.5 in production (which is currently most of them).